examples/canvas/fbo_canvas.py

```diff
@@ -18,6 +17,7 @@
 from kivy.core.window import Window
 from kivy.animation import Animation
 from kivy.factory import Factory
+__all__ = ('FboFloatLayout', )
```

Comment:
not sure if this is ok, or may raise issues e.g. with trying to import something else than `FboFloatLayout` i.e. after you move `__all__` to the bottom, this could be possible, I think? @dessant ?

    from fbo_canvas import Window, Animation, ...